### ISSUE TITLE
Test against WP 5.4

### ISSUE THREAD

**Issue by jeffpaul** [10up/simple-podcasting#80]
**Is your enhancement related to a problem? Please describe.**
Once WordPress 5.4 is released, we'll want to test Simple Podcasting to see if any incompatibility issues arise.

**Describe the solution you'd like**
- [x] test Simple Podcasting on WP 5.4
- [ ] open issues for any incompatibilities noted in testing
- [ ] resolve issues identified in testing
- [x] bump "tested up to" version
- [ ] if code changes needed due to incompatibilities, ship a plugin release, otherwise use "Plugin asset/readme update" action to update "tested up to" version on .org repo

**Designs**
n/a

**Describe alternatives you've considered**
none

**Additional context**
none

**Comment by Waka867** [comment]
@jeffpaul I can go through this one

**Comment by jeffpaul** [comment]
@Waka867 awesome, many thanks!

**Comment by jeffpaul** [comment]
@Waka867 how did your testing go with Simple Podcasting and WordPress 5.3?

**Comment by Waka867** [comment]
@jeffpaul Sorry haven't gotten to this yet due to some stuff coming up over the holidays. Will try to hit this tomorrow am!

**Comment by Waka867** [comment]
@jeffpaul Sorry for the delay, I went through Simple Podcasting and found a problem that I've opened an issue for [here](https://github.com/10up/simple-podcasting/issues/81#issue-549051879)


Here are my notes from the testing I did. Other than that issue, I found no other problems with the plugin.

**Notes**
Adding podcasts functions properly
Edit podcast page all fields submit properly
XMl RSS Feed doc generates correctly with the right info
Feed URL properly lists each podcast item as they are added
URL validated with podba.se 
Gutenberg block properly usable
On post view, preview, and blog listing pages, gutenberg podcast block does not play, no console errors shown. Time shows 0:00/0:00 and podcast doesn't load, file plays when added on the edit page. Adding conventional audio block works fine.
Server php/apache error logs show no errors on day or even week of test
Submitted issue #81
